apps/vpnmanager/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "vpnmanager",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "build-ts": "tsc --project tsconfig.server.json && tsc-alias -p tsconfig.server.json",
+    "process-new-hires": "node dist/src/lib/processNewHires.js"
+  },
+  "dependencies": {
+    "@babel/core": "^7.23.6",
+    "@babel/preset-react": "^7.23.3",
+    "@commons-ui/core": "workspace:*",
+    "@commons-ui/next": "workspace:*",
+    "@emotion/cache": "^11.11.0",
+    "@emotion/react": "^11.11.1",
+    "@emotion/server": "^11.11.0",
+    "@emotion/styled": "^11.11.0",
+    "@mui/material": "^5.14.20",
+    "@mui/utils": "^5.14.20",
+    "@sentry/nextjs": "^7.105.0",
+    "@svgr/webpack": "^8.1.0",
+    "@types/jest": "^29.5.12",
+    "googleapis": "^133.0.0",
+    "jest": "^29.7.0",
+    "next": "14.1.3",
+    "react": "^18",
+    "react-dom": "^18",
```

Comment:
Use the same exact version pinning as in charterafrica, codeforafrica, etc. We don't want to end up with different versions.